internal/pkg/fileutil/fileutil.go

```diff
@@ -0,0 +1,44 @@
+// Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+// or more contributor license agreements. Licensed under the Elastic License;
+// you may not use this file except in compliance with the Elastic License.
+
+package fileutil
+
+import (
+	"errors"
+	"io/fs"
+	"os"
+	"time"
+)
+
+// FileExists returns true if file/dir exists
+func FileExists(fp string) (ok bool, err error) {
+	if _, err := os.Stat(fp); err == nil {
+		ok = true
+	} else if errors.Is(err, fs.ErrNotExist) {
+		err = nil
```

Comment:
The linter is complaining because this instance of `err` is the one defined in the if statement, not the named return.
I found that using explicit returns instead of named returns makes it easier to avoid these types of issues